<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkPage</class>
 <widget class="QWidget" name="NetworkPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>1127</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Network&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line9_3">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Online Status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="dynamicOnlineCheckBox">
        <property name="toolTip">
         <string>Select to determine the online status dynamically (assume online if unchecked)</string>
        </property>
        <property name="text">
         <string>Dynamic Online Status Determination</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Downloads</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Download directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="E5PathPicker" name="downloadDirPicker" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="toolTip">
           <string>Enter the download directory (leave empty to use the default location)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="requestFilenameCheckBox">
        <property name="toolTip">
         <string>Select to ask the user for a download filename</string>
        </property>
        <property name="text">
         <string>Request name of downloaded file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="cleanupGroup">
        <property name="title">
         <string>Download Manager Cleanup Policy</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="cleanupNeverButton">
           <property name="toolTip">
            <string>Select to never cleanup automatically</string>
           </property>
           <property name="text">
            <string>Never</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="cleanupExitButton">
           <property name="toolTip">
            <string>Select to cleanup upon exiting</string>
           </property>
           <property name="text">
            <string>When exiting the application</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="cleanupSuccessfulButton">
           <property name="toolTip">
            <string>Select to cleanup after a successful download</string>
           </property>
           <property name="text">
            <string>When download finished successfully</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="proxyGroup">
     <property name="toolTip">
      <string>Select to use a web proxy</string>
     </property>
     <property name="title">
      <string>Network Proxy</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="noProxyButton">
        <property name="toolTip">
         <string>Select to not use a network proxy</string>
        </property>
        <property name="text">
         <string>Do not use proxy</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="systemProxyButton">
        <property name="toolTip">
         <string>Select to use the system proxy configuration</string>
        </property>
        <property name="text">
         <string>Use system proxy configuration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="manualProxyButton">
        <property name="toolTip">
         <string>Select to use an application specific proxy configuration</string>
        </property>
        <property name="text">
         <string>Manual proxy configuration:</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Manual proxy settings</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>HTTP-Proxy</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Hostname:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="httpProxyHostEdit">
              <property name="toolTip">
               <string>Enter the name of the HTTP proxy host</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="httpProxyPortSpin">
              <property name="toolTip">
               <string>Enter the HTTP proxy port</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="4">
             <widget class="QCheckBox" name="httpProxyForAllCheckBox">
              <property name="toolTip">
               <string>Select to use the HTTP proxy for all</string>
              </property>
              <property name="text">
               <string>Use this proxy for all protocols</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>HTTPS-Proxy</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Hostname:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="httpsProxyHostEdit">
              <property name="toolTip">
               <string>Enter the name of the HTTPS proxy host</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="httpsProxyPortSpin">
              <property name="toolTip">
               <string>Enter the HTTPS proxy port</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>443</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>FTP-Proxy</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Proxy Type:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="3">
             <widget class="QComboBox" name="ftpProxyTypeCombo">
              <property name="toolTip">
               <string>Select the type of the FTP proxy</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Hostname:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="ftpProxyHostEdit">
              <property name="toolTip">
               <string>Enter the name of the FTP proxy host</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QSpinBox" name="ftpProxyPortSpin">
              <property name="toolTip">
               <string>Enter the FTP proxy port</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>21</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>User Name:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="3">
             <widget class="QLineEdit" name="ftpProxyUserEdit">
              <property name="toolTip">
               <string>Enter the user name for the proxy authentication</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Password:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="3">
             <widget class="QLineEdit" name="ftpProxyPasswordEdit">
              <property name="toolTip">
               <string>Enter the password for the proxy authentication</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Account:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="3">
             <widget class="QLineEdit" name="ftpProxyAccountEdit">
              <property name="toolTip">
               <string>Enter the account info for the proxy authentication</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Exceptions:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="exceptionsEdit">
          <property name="toolTip">
           <string>Enter host names or IP-addresses for which the proxy is to be circumvented separated by ',' (wildcards * or ? may be used)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="clearProxyPasswordsButton">
        <property name="toolTip">
         <string>Press to clear the saved passwords for the Http(s) proxy</string>
        </property>
        <property name="text">
         <string>Clear HTTP(S) Proxy Passwords</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>571</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dynamicOnlineCheckBox</tabstop>
  <tabstop>downloadDirPicker</tabstop>
  <tabstop>requestFilenameCheckBox</tabstop>
  <tabstop>cleanupNeverButton</tabstop>
  <tabstop>cleanupExitButton</tabstop>
  <tabstop>cleanupSuccessfulButton</tabstop>
  <tabstop>noProxyButton</tabstop>
  <tabstop>systemProxyButton</tabstop>
  <tabstop>manualProxyButton</tabstop>
  <tabstop>httpProxyHostEdit</tabstop>
  <tabstop>httpProxyPortSpin</tabstop>
  <tabstop>httpProxyForAllCheckBox</tabstop>
  <tabstop>httpsProxyHostEdit</tabstop>
  <tabstop>httpsProxyPortSpin</tabstop>
  <tabstop>ftpProxyTypeCombo</tabstop>
  <tabstop>ftpProxyHostEdit</tabstop>
  <tabstop>ftpProxyPortSpin</tabstop>
  <tabstop>ftpProxyUserEdit</tabstop>
  <tabstop>ftpProxyPasswordEdit</tabstop>
  <tabstop>ftpProxyAccountEdit</tabstop>
  <tabstop>exceptionsEdit</tabstop>
  <tabstop>clearProxyPasswordsButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>manualProxyButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>httpProxyForAllCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>httpProxyForAllCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
